relational/table.py

```diff
@@ -260,3 +262,159 @@ def rows(self, proj):
         self._square_feet(result, proj)
         self._bedroom_info(result, proj)
         return result
+
+
+class ProjectStatusHistory(Table):
+    _PPTS_ENT_CODES = {'ENV', 'AHB', 'COA', 'CUA', 'CTZ', 'DNX', 'ENX',
+                       'OFA', 'PTA', 'SHD', 'TDM', 'VAR', 'WLS'}
+
+    TOP_LEVEL_STATUS = 'top_level_status'
+    START_DATE = 'start_date'
+    END_DATE = 'end_date'
+    SEQUENTIAL = 'sequential'
```

Comment:
doesn't appear to be used here

Reply:
Good point and removed, given that we've decided for the time being everythign is sequential probably don't need this